index.html

```diff
@@ -45,5 +47,247 @@ <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                <h2>Tonic</h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <div class="work4">
+
+            <div>
+                <a>
+                    <img src="img/work4.png" alt="work image">
+                </a>
+            </div>
+            <div class="work1-title">
+                <h2>Multi-Post <br>
+                    Stories<br>
+                </h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <section class="about-myself">
+            <div class="about-myself-container">
+                <div class="title">
+                    <h2>About Myself</h2>
+                </div>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), As requested by the previous reviewer, The `About Myself` section text  color and the section border-right-radius does not match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) :blush: 

| Your Design |Figma Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/229325139-e618d84b-152c-4789-ab19-69e9641e302d.png)| ![image](https://user-images.githubusercontent.com/92303891/229325057-171e1988-49a0-4af8-bdc2-d3f2371f516e.png)|